communication/youtube/youtube-guidelines.md

```diff
@@ -56,191 +40,222 @@ Hours, will be shared with the appropriate point(s) of contact.
 
 ### Uploading Guidelines for Collaborators
 
-**NOTE:** If you're using a G Suite account you may need to [loosen the
-permissions in your YouTube settings]. If you have any questions reach out to
-the [YouTube admins] or [SIG Contributor Experience].
+**NOTE:** If you're using a Google Workspace account (formerly known as G Suite)
+you may need to [update the permissions in your YouTube settings][]. If you have
+any questions, reach out to the [YouTube admins][] or
+[SIG Contributor Experience][]. You may need to reach out to someone at your
+organization if you do not have access to Google Workspace Admin permissions.
 
-**NOTE:** Both public and private steering meeting recordings should be made public.
+**NOTE:** Both public and private steering meeting recordings should be made
```

Comment:
I don't think private meetings are public. @mrbobbytables is this correct?

Reply:
Private recordings should be made public. 👍 steering only records the portions of the meeting that should be public. It seems a little weird, but thats the name that the meeting comes over with >_<